### ISSUE TITLE
End-of-Match Statistic: Most Blocks Removed

### ISSUE THREAD

**Issue by arcadeboss** [PGMDev/PGM#1196]
Along with Most Kills, Longest Kill Streak, etc... please add statistic that shows the player with the most blocks removed and the number of blocks they removed.

A simple feature that would be very nice for the defenders, who rarely get accolades.

Would be very interesting to see this data, especially on longer/larger maps.

If this statistic isn't already tracked, thanks to Pablo, can share this reference to Stratus, which has implementation for tracking "destroyables".   In this file, and others:

https://github.com/StratusNetwork/projectares/blob/master/PGM/src/main/java/tc/oc/pgm/highlights/HighlightListener.java#L67

Idea proposed in Discord Feedback, and had multiple supporters.

**Comment by Pablete1234** [comment]
That reference is of a completely different feature, its MVP calculation logic. Has notthing to do with the acutal tracking.

**Comment by arcadeboss** [comment]
> That reference is of a completely different feature, its MVP calculation logic. Has notthing to do with the acutal tracking.

Yes, that is correct.   However, link to Ares was shared as Ares has the actual tracking as well.

**Comment by Pablete1234** [comment]
Code used to use player facets for stat tracking, those do not exist in this codebase, and stat tracking is a completely different system

The overlap of what could even be reusable is near 0, it simply has to be coded in this codebase from scratch. Not that it's a difficult feature, simply that there's little to nothing that can simply be reused, hence why the reference isn't useful.

**Comment by arcadeboss** [comment]
Excellent, thank you!

**Comment by arcadeboss** [comment]
Saw discussion about the blocks destroyed feature of battle pass and it might also make sense to include similar logic to not count easy to break blocks.  A qualitative scoring component as well would be welcome, but the raw number of blocks broken would still be a desired statistic for selected player.
